projects/packages/classic-theme-helper/_inc/lib/tonesque.php

```diff
@@ -0,0 +1,278 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+/**
+ * Tonesque
+ * Grab an average color representation from an image.
+ *
+ * @author Automattic
+ * @author Matias Ventura
+ * @package automattic/jetpack
+ */
+
+/**
+ * Color representation class.
+ */
+class Tonesque {
+	/**
+	 * Image URL.
+	 *
+	 * @var string
+	 */
+	private $image_url = '';
+	/**
+	 * Image identifier representing the image.
+	 *
+	 * @var null|object
+	 */
+	private $image_obj = null;
+	/**
+	 * Color code.
+	 *
+	 * @var string
+	 */
+	private $color = '';
+
+	/**
+	 * Constructor.
+	 *
+	 * @param string $image_url Image URL.
+	 */
+	public function __construct( $image_url ) {
+		if ( ! class_exists( 'Jetpack_Color' ) ) {
+			require_once JETPACK__PLUGIN_DIR . '/_inc/lib/class.color.php';
```

Comment:
I notice that the use of JETPACK__PLUGIN_DIR was responsible for one of the phan reported errors (`UndefError PhanUndeclaredConstant Reference to undeclared constant \JETPACK__PLUGIN_DIR. This will
    | cause a thrown Error in php 8.0+.`)
I see elsewhere (for example the Sync package) that we check if it is defined first (`if ( defined( 'JETPACK__PLUGIN_DIR' ) ) {`).

Reply:
Good catch! I fixed it.